crawler_mcp/core/vectors.py

```
@@ -345,16 +346,16 @@ async def search_similar(
-                },  # Dynamic ef for optimal speed/accuracy
+                search_params=SearchParams(
+                    hnsw_ef=ef_value
+                ),  # Dynamic ef for optimal speed/accuracy
             )
 
             # Extract results - handle different return types from query_points
             results: list[Any] = []
             if hasattr(query_response, "points"):
                 results = query_response.points
-            elif isinstance(query_response, tuple | list) and len(query_response) > 0:
+            elif isinstance(query_response, (tuple, list)) and len(query_response) > 0:
```

Ruff (0.12.2)

Use `X | Y` in `isinstance` call instead of `(X, Y)`

Convert to `X | Y`

(UP038)